### ISSUE TITLE
goctl model mysql datasource 生成 Model 代码多引用依赖问题

### ISSUE THREAD

**Issue by calvinit** [zeromicro/go-zero#3825]
**Describe the bug**

如果不生成带缓存的代码，xxx_model_gen.go 不应该 import 进来 "github.com/zeromicro/go-zero/core/stores/sqlc" 依赖！

**To Reproduce**

1. The code is

   ```shell
   goctl model mysql datasource --url 'xxx' -t '*' --style go_zero -d model
   ```

2. The error is

   ```
   Error: Unused import "github.com/zeromicro/go-zero/core/stores/sqlc"
   ```

**Expected behavior**

**Screenshots**

**Environments (please complete the following information):**
 - OS: MacOS Sonoma 14.2.1
 - go-zero version 1.6.1
 - goctl version 1.6.1 darwin/amd64

**Comment by kesonan** [comment]
This will be fixed on pr #3828

**Comment by kevwan** [comment]
Fixed, closing. Reopen if further problem. Thanks!
